cli/README.md

```diff
@@ -11,9 +14,69 @@ npx link
 When you've made a change to the CLI and want to test it out, you can rerun the build and link steps.
 This will make the CLI available on your local `node_modules` path.
 
-Type `npx calm` into your terminal, and you should see the help text printed out.
-
 `npx link` uses the `link` package to symlink the `calm` executable in `node_modules/.bin` to your locally-built CLI.
 
 Note: you can also use `npm link` but this installs to your global package registry.
 This will make the executable available as just `calm`, but will pollute your global NPM profile and may require `sudo` depending on your OS.
+
+### Using the CLI
+Type `npx calm` into your terminal, and you should see the help text printed out.
+
+```shell
+% npx calm
+Usage: calm [options] [command]
+
+A set of utilities for interacting with CALM
+
+Options:
+  -V, --version        output the version number
+  -h, --help           display help for command
+
+Commands:
+  visualize [options]  Produces an SVG file representing a visualization of the CALM Specification.
+  generate [options]   Generate an instantiation from a CALM pattern file.
+  validate [options]
+  help [command]       display help for command
+```
+
+### Visualizing the CALM schema
```

Comment:
just did an update that changed this in my most recent PR, so now there's also a pattern option:

```shell
➜  cli git:(visualize-patterns) ✗ calm visualize --help
Usage: calm visualize [options]

Produces an SVG file representing a visualization of the CALM Specification.

Options:
  -i, --instantiation <file>  Path to an instantiation of a CALM pattern.
  -p, --pattern <file>        Path to a CALM pattern.
  -o, --output <file>         Path location at which to output the SVG. (default: "calm-visualization.svg")
  -v, --verbose               Enable verbose logging. (default: false)
  -h, --help                  display help for command
```

Reply:
Is it worth merging this in then making the update to your PR to reflect the change?

Reply:
my PR has been merged already, although I could raise another if you're just wanting to get this merged in